frontend/src/__tests__/unit/utils/pipelines/convertPeriodicTime.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { convertPeriodicTimeToSeconds } from '~/concepts/pipelines/content/createRun/submitUtils';
+
+describe('Convert periodic time to seconds', () => {
+  it('should convert hours to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('5Hour')).toBe(5 * 3600);
+  });
+
+  it('should convert minutes to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('6Minute')).toBe(6 * 60);
+  });
+
+  it('should convert days to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('221Day')).toBe(221 * 86400);
+  });
+
+  it('should convert weeks to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('12Week')).toBe(12 * 7 * 24 * 60 * 60);
+  });
+
+  it('should default to 0 seconds for unrecognized units', () => {
```

Comment:
Include a test for NaN value or prevent NaN values from being a return value. I would just check that the value is not NaN before you return